lib/routes/anime1/search.ts

```diff
@@ -0,0 +1,63 @@
+import { Route } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    path: 'search/:keyword',
+    name: 'Search',
+    url: 'anime1.me',
+    maintainers: ['cxheng315'],
+    example: '/anime1/search/神之塔',
+    categories: ['anime'],
+    parameters: {
+        keyword: 'Anime1 Search Keyword',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['anime1.me/?s=:keyword'],
```

Comment:
`source` can't match any search parameters

Reply:
Sorry, I don't quite follow this. The URL should look like `https://anime1.me/?s=123`, so the source should work

Reply:
`s` is the [search parameter](https://nodejs.org/api/url.html#urlsearchparams) which `source` can't match

Reply:
will `anime1.me/?s=${encodeURIComponent(:keyword)}` works?

Reply:
> `source` can't match any search parameters

No.

Reply:
any suggestion? or keep `anime1.me` only

Reply:
You can remove the rule.